renderdoc/driver/gl/wrappers/gl_emulated.cpp

```diff
@@ -3314,6 +3334,11 @@ void GLDispatchTable::EmulateRequiredExtensions()
     EMULATE_FUNC(glVertexArrayElementBuffer);
     EMULATE_FUNC(glVertexArrayVertexBuffers)
   }
+
+  if(GLCoreVersion < 45)
+  {
+    EMULATE_FUNC(glInvalidateNamedFramebufferData);
```

Comment:
I think this should go in the above `if(!HasExt[ARB_direct_state_access])` because even an implementation that's core 4.4 may implement `ARB_direct_state_access` and `ARB_direct_state_access` provides `glInvalidateNamedFramebufferData`. (It only provides the function if `ARB_invalidate_subdata` is supported too, but that should fall out in the wash with the emulation so we can just check on `ARB_direct_state_access` - if the invalidate extension isn't supported then we won't be able to emulate anyway since the `EXT` variant is GLES only).